.github/workflows/CI.yml

```
@@ -0,0 +1,69 @@
+name: CI
+
+on:
+  pull_request:
+    branches:
+      - 'main'
+
+jobs:
+  pre-commit:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-python@v5
+      - uses: pre-commit/action@2c7b3805fd2a0fd8c1884dcaebf91fc102a13ecd #v3.0.1
+
+  pypi-build:
+    name: Build package for PyPI
+    if: github.repository == 'access-nri/model-config-tests'
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Setup Python 3.11
+        uses: actions/setup-python@v5
+        with:
+          python-version: 3.11
```

This needs to be in quotes because otherwise it interprets it as a floating point rather than the required `string`:
```suggestion
          python-version: '3.11'
```